src/components/CompareResults/beta/ResultsView.tsx

```diff
@@ -11,9 +12,9 @@ function ResultsView() {
       <Grid container alignItems='center' justifyContent='center'>
         <Grid item xs={10}>
           {/* TODO PCF-223 */}
-          Beta version
         </Grid>
         <Grid item xs={12}>
+          <InputsReplacement />
```

Comment:
And this line could be moved to replace the todo PCF-223 comment inside the grid